packages/flame/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Refer to the [CONTRIBUTING guide](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md) for more info.
 
+## [Unreleased]
+
+### Fixed
+
+- Typecast Button to bypass weird automatic type generation shenanigans
```

Comment:
Best changelog entry we've ever had lol. Include PR number and let's remove "shenanigans" please 😉